README.md

```diff
@@ -28,94 +28,86 @@ Aligned works with EigenLayer to leverage ethereum consensus mechanism for ZK pr
 
 - [Rust](https://www.rust-lang.org/tools/install)
 
+To install the batcher client to send proofs in the testnet, run: 
+
+```bash
+make install_batcher_client
+```
+
 ### Run
 
 #### SP1 proof
 
 The SP1 proof needs the proof file and the vm program file.
 
 ```bash
-pushd batcher/client/ ; \
-cargo run --  \
+batcher-client \
 --proving_system <SP1|GnarkPlonkBn254|GnarkPlonkBls12_381|Groth16Bn254> \
 --proof <proof_file> \
 --vm_program <vm_program_file> \
 --conn wss://batcher.alignedlayer.com \
---proof_generator_addr [proof_generator_addr] ;
-popd
+--proof_generator_addr [proof_generator_addr]
 ```
 
 **Example**
 
 ```bash
-pushd batcher/client/ ; \
-cargo run -- \
+batcher-client \
 --proving_system SP1 \
 --proof test_files/sp1/sp1_fibonacci.proof \
 --vm_program test_files/sp1/sp1_fibonacci-elf \
---conn wss://batcher.alignedlayer.com ;
-popd
+--conn wss://batcher.alignedlayer.com
 ```
 
 ```bash
-pushd batcher/client/ ; \
-cargo run -- \
+batcher-client \
 --proving_system SP1 \
 --proof test_files/sp1/sp1_fibonacci.proof \
 --vm_program test_files/sp1/sp1_fibonacci-elf \
---conn wss://batcher.alignedlayer.com ;
-popd
+--conn wss://batcher.alignedlayer.com
```

Comment:
same here

Reply:
It was removed in this [commit](https://github.com/yetanotherco/aligned_layer/pull/283)